tests/plugins/test_plugin.py

```diff
@@ -23,10 +23,11 @@
 from airflow.executors.base_executor import BaseExecutor
 # Importing base classes that we need to derive
 from airflow.hooks.base_hook import BaseHook
-from airflow.models.baseoperator import BaseOperator, BaseOperatorLink
+from airflow.models.baseoperator import BaseOperator
 # This is the class you derive to create a plugin
 from airflow.plugins_manager import AirflowPlugin
 from airflow.sensors.base_sensor_operator import BaseSensorOperator
+from tests.utils.extra_link_operators import AirflowLink, AirflowLink2, GithubLink, GoogleLink
```

Comment:
I'm not 100% happy with this module name - not sure I have a better one, but these aren't "utils"

Reply:
Changed